src/decimal.ts

```diff
@@ -0,0 +1,160 @@
+import { StateConverterOptionsWithContext } from "./converter";
+import { DecimalOptions } from "./converters";
```

Comment:
This causes a circular import. That's not critical in this case apparently, but still something we prefer to avoid if we can do so easily. So I suggest you move DecimalOptions into this file and import it in converters.ts instead.

Reply:
It may be needed to include this file in index.ts to get it be importable.